Userland/Libraries/LibWeb/HTML/SessionHistoryEntry.h

```diff
@@ -26,7 +27,12 @@ enum class ScrollRestorationMode {
 };
 
 // https://html.spec.whatwg.org/multipage/history.html#session-history-entry
-struct SessionHistoryEntry {
+struct SessionHistoryEntry final : public JS::Cell {
+    JS_CELL(SessionHistoryEntry, JS::Cell);
+
+    void visit_edges(Cell::Visitor&) override;
```

Comment:
`struct` members are public by default, so having a separate public section below is confusing. Arguably this is complex enough to be a class now.